tilelang/autodd.py

```
@@ -0,0 +1,1170 @@
+        return applied
+
+    @override
+    def task_generator(self) -> Iterable[Task]:
+        for task in self.generator():
+            new_tree, _ = apply_rewrites(self.tree, task.applied)
+            try:
+                new_tree = deepcopy(new_tree)
+                ast.fix_missing_locations(new_tree)
+                source = ast.unparse(new_tree)
+            except Exception as _:
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)